src/main/java/makaroshyna/onlinebookstore/model/Book.java

```diff
@@ -0,0 +1,29 @@
+package makaroshyna.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```

Comment:
there is a recommendation and a warning in logs to not use `@Data` with entity classes